test/junit/scala/concurrent/duration/DurationTest.scala

```diff
@@ -0,0 +1,27 @@
+package scala.concurrent.duration
```

Comment:
I agree the tests fit better in the existing set of tests for Duration, where they can eventually be migrated together with the others. Shall we go with that option then?
I have to say finding the right place to test this functionality was definitely the hardest part of this contribution.